src/transformers/CommonUtils.h

```diff
@@ -9,6 +9,7 @@
 
 #include "graph/ChcGraph.h"
 #include "Witnesses.h"
+#include "TransformationUtils.h"
```

Comment:
Why do you need this include?

Reply:
This one is needed for `getVariablesFromEdge` function which is used in EdgeTranslate

Reply:
Then it should be included in `.cc` file, not in the header.